pkg/time/time.go

```diff
@@ -64,3 +65,34 @@ func FromString(input string) (time.Time, error) {
 		"failed to convert %s to time", input,
 	)
 }
+
+// FormatNames are the valid time format names accepted by this package.
+var FormatNames = []string{
+	"StampMilli",
+	"RFC3339",
+	"RFC3339Milli",
+	"RFC3339Micro",
+	"RFC3339Nano",
+	"RFC1123Z",
+}
+
+// TimeFormatNameToLayout returns the time format layout for the time format
+// name.
+func TimeFormatNameToLayout(name string) string {
```

Comment:
```suggestion
func FormatNameToLayout(name string) string {
```

Reply:
Thanks. I forgot to run `staticcheck` locally, should now be fixed.